jquery.highlight-within-textarea.css

```diff
@@ -29,10 +29,8 @@
 .hwt-input {
 	display: block !important;
 	position: relative !important;
-	margin: 0;
-	padding: 0;
+	margin: 0 !important;
```

Comment:
Why was margin left behind?

Reply:
If the original textarea had any margin set, it would be applied inside the .hwt-container, instead we want to add that amount of margin to the actual .hwt-container (so that it feels like the original textarea element) and remove any margin from the textarea inside